shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/test/java/org/apache/shardingsphere/proxy/config/util/DataSourceParameterConverterTest.java

```diff
@@ -44,11 +44,7 @@ public void assertGetDataSourceConfigurationMap() {
     }
     
     private DataSourceParameter crateDataSourceParameter() {
-        DataSourceParameter result = new DataSourceParameter();
-        result.setUrl("jdbc:mysql://localhost:3306/demo_ds");
-        result.setUsername("root");
-        result.setPassword("root");
-        return result;
+        return new DataSourceParameter("jdbc:mysql://localhost:3306/demo_ds", "root", "root", null, null, null, null, null, null, null);
```

Comment:
ten parameters in a constructor, this is not that  elegant

Reply:
I will change in next PR